src/Orchard.Specs/Widgets.feature

```diff
@@ -12,12 +12,27 @@ Scenario: I can edit a default layer
     When I fill in
             | name | value |
             | LayerPart.Description | This is the default layer. |
-        And I hit "Save Draft"
+        And I hit "Publish"
         And I am redirected
     Then I should see "Your Layer has been saved"
     When I follow "Edit"
     Then I should see "<textarea[^>]*>\s*This is the default layer.\s*</textarea>"
 
+#Scenario: I can save draft a default layer
```

Comment:
Removed?

Reply:
This file I comment three scenarios. All of three are about edit or add widget and **Save as draft**. But Orchard UI has only **Publish** button not Save Draft. I will fix this with add more scenario same as I commented but with Publish button.